.code-samples.meilisearch.yaml

```
@@ -3,6 +3,12 @@
 # the documentation on build
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
+filtering_guide_nested_1: |-
+  with data: { "index": "movies", "q": "thriller", "filter": "rating.users >= 90" }
```

```suggestion
  with data: { "index": "movie_ratings", "q": "thriller", "filter": "rating.users >= 90" }
```

https://github.com/meilisearch/documentation/blob/8b28cd20eb2668fac8fce2bba9f70a625ac2b73b/.code-samples.meilisearch.yaml#L361-L368